collect_app/src/main/java/org/odk/collect/android/preferences/GeneralKeys.java

```diff
@@ -95,7 +96,7 @@
     private static HashMap<String, Object> getHashMap() {
         HashMap<String, Object> hashMap = new HashMap<>();
         // odk_server_preferences.xmll
-        hashMap.put(KEY_SERVER_URL,                 Collect.getInstance().getString(R.string.default_server_url));
+        hashMap.put(KEY_SERVER_URL,                 TranslationHandler.getString(Collect.getInstance(), R.string.default_server_url));
```

Comment:
All the values in here should be in `unstranslated.xml` right? I think we should probably just use `getString` directly here.